docs/installation/java-installation.rst

```diff
@@ -7,11 +7,13 @@ Prerequisites: :ref:`presto-admin-installation-label` and :ref:`presto-admin-con
 
 The Oracle Java 1.8 JRE (64-bit), update 45 or higher, is a prerequisite for Presto. If a suitable 64-bit version of Oracle Java 8 is already installed on the cluster, you can skip this step.
 
-The ``presto-admin`` tool simplifies installing Java 8 on all nodes in the Presto cluster. To install the Oracle Java 1.8 (64-bit) RPM for Linux using ``presto-admin``, first download `Oracle Java 8 <http://java.com/en/download/linux_manual.jsp>`_ and copy it to a location accessible by ``presto-admin``. Next, run the following command to install Java 8 on each node in the Presto cluster.
+The simplest way to install Java 8 on all nodes in the Presto cluster is to use ``presto-admin``. To install the Oracle Java 1.8 (64-bit) RPM for Linux using ``presto-admin``, first download `Oracle Java 8 <http://java.com/en/download/linux_manual.jsp>`_ and copy it to a location accessible by ``presto-admin``. Next, run the following command to install Java 8 on each node in the Presto cluster.
 ::
```

Comment:
Suggestion:
To install the Oracle Java 1.8 (64-bit) RPM for Linux using presto-admin:
1. Download...
2. Copy.....
3. Run....
    (command)